### PR TITLE
20220312 homebridge - master branch - PR 1 of 2

### PR DESCRIPTION
**Pull request by Paraphraser**
Changes image from `oznu/homebridge:no-avahi-arm32v6` to
`oznu/homebridge:latest`:

* `homebridge:latest` pulls `arm` or `arm64` as appropriate
(ie Buster/Bullseye)

* IOTstack defaults to running the container in host mode (as per the
[homebridge documentation](https://github.com/oznu/docker-homebridge/wiki/Homebridge-on-Raspberry-Pi)),
therefore the container can see multicastDNS traffic, therefore
"no-avahi" was never really appropriate. Further, current
[homebridge documentation](https://github.com/oznu/docker-homebridge/wiki)
says:

	> Note: Avahi is now disabled by default in all Alpine builds as
Homebridge no longer requires it. If you have a plugin that requires
avahi, it can enabled by setting the environment variable ENABLE_AVAHI=1.

	`homebridge:latest` pulls an Alpine-based image so the above note
is applicable.

Replaced UI port placeholder with fixed port 8581. This is in line with
the [homebridge documentation](https://github.com/oznu/docker-homebridge/wiki/Homebridge-on-Raspberry-Pi#managing-homebridge).

Removed `build.py` from the homebridge template. The script was
throwing errors, was not substituting the port placeholder with a
sensible default, and was not offering any mechanism to replace the
placeholder with a proper port number.

Added basic documentation.

Signed-off-by: Phill Kelley <34226495+Paraphraser@users.noreply.github.com>

### CODE REVIEW

**Review by ukkopahis — APPROVED** [review]
Nice work.

**Review by ukkopahis — CHANGES_REQUESTED** [review]
Also update the port for homebridge on the table in `docs/Basic_setup/Default-Configs.md`?

ps. you can probably add the edit easily directly from the browser using [this link](https://github.com/Paraphraser/IOTstack/edit/20220312-homebridge-master/docs/Basic_setup/Default-Configs.md).

**Comment by Paraphraser** [comment]
Done. Thanks for pointing that out.

**Review by ukkopahis — APPROVED** [review]

File: docs/Containers/Homebridge.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+* If you are running new menu (master branch, the default), environment variables are kept inline in `docker-compose.yml`.
+* If you are running old menu (old-menu branch), environment variables are at the path:
+
+	```
+	~/IOTstack/services/homebridge/homebridge.env
+	```
+
+In either case, you apply changes by editing the relevant file (`docker-compose.yml` or `homebridge.env`) and then:
+
+```bash
```
Comment (ukkopahis) [inline docs/Containers/Homebridge.md:24]: In regards to #547, could you change this from `bash` to `console`?

Comment (Paraphraser) [reply docs/Containers/Homebridge.md]: Done.

**Review by Slyke — APPROVED** [review]

**Pull request merged**